src/Evolution/Executables/GrMhd/GhValenciaDivClean/GhValenciaDivCleanBase.hpp

```diff
@@ -761,6 +762,10 @@ struct GhValenciaDivCleanTemplateBase<
           use_dg_element_collection>,
       evolution::dg::Actions::ApplyBoundaryCorrectionsToTimeDerivative<
           system, volume_dim, false, use_dg_element_collection>,
+      tmpl::conditional_t<
+          UseControlSystems, tmpl::list<>,
+          Actions::MutateApply<
+              grmhd::GhValenciaDivClean::subcell::ZeroMhdTimeDerivatives>>,
```

Comment:
This adds the `ZeroMhdTimeDerivatives` to all GhGrmhd execs *except* the BNS... was that intended?

Reply:
🤦  I'll submit a PR to fix